### ISSUE TITLE
Datepicker component keyboard navigation

### ISSUE THREAD

**Issue by Akshat55** [carbon-design-system/carbon-components-angular#3179]
* Tab should be used for focusing on calendar today's date
* If the calendar is open and user presses tab, the calendar should close and focus on next focus-able element

To resolve this, we will likely need to add keyboard event listener to all of the elements in the calendar.
